lessons/module-1/js-dot-bracket-notation.md

```diff
@@ -147,6 +147,36 @@ Remember: a variable represents some other value, and that value could be reassi
 We must use bracket notation whenever we are accessing an object's property using a *variable* or when the property's key is a *number* or includes a *symbol* or is *two words with a space*.
 </section>
 
+Take a moment to read through this code:
+```js
```

Comment:
Love the changes you made! I think this is a great chunk of code for students to read through and predict. 

One consideration to make is that (I think) this code can only ever return 'Sorry, we aren't serving <food item here> today.', since the `.includes` method can't compare objects. For example: `[{name: 'Quiche', price: '6.49', type: 'breakfast'}].includes({name: 'Quiche', price: '6.49', type: 'breakfast'})` will return `false`. 

I'm not sure if this matters at all! Students are still getting AMAZING practice reading through code and deciphering how bracket notation resolves vs how dot notation resolves with this example. 

My vote is probably to just leave it - but I did want to bring it up in case y'all think this will be confusing for students.

Reply:
Love it!